pylib/Tools/Build/Autotools.py

```diff
@@ -197,6 +199,16 @@ def execute(self, log, keyvals, testDef):
                     # this is a multistep operation, and so we need to
                     # retain the output from each step in the log
                     log['autogen'] = (stdout, stderr)
+
+                    # Use mpi4py to get MPI information for IUDatabase plugin
+                    log['mpi_name'] = MPI.get_vendor()[0]
```

Comment:
IIRC, mpi4py has to be configured and built against an MPI, yes? And so the macros are setup at time of compiling mpi4py, which means they will only tell you information about the MPI which mpi4py was built against - it doesn't tell you the version of MPI you have in your path at the moment.

Is that not true? I don't think you can just "point" mpi4py to any MPI at runtime and have it work.